client/src/components/Login.js

```diff
@@ -19,14 +19,20 @@ const Login = props => {
     event.preventDefault();
     loginUser(email, password)
       .then(data => {
-        const token = data.token;
+        if (Object.keys(data).length === 0) {
```

Comment:
Good. I would probably just use `if(data.token)` to check for the presence of the token on the response and use the `else` to set the error. It's more reliable than assuming that the response here is an empty object (even though we happened to code it that way in the `loginUser` function.